components/script_bindings/webidls/DedicatedWorkerGlobalScope.webidl

```diff
@@ -5,6 +5,7 @@
 // https://html.spec.whatwg.org/multipage/#dedicatedworkerglobalscope
 [Global=(Worker,DedicatedWorker), Exposed=DedicatedWorker]
 interface DedicatedWorkerGlobalScope : WorkerGlobalScope {
+  readonly attribute DOMString name;
```

Comment:
This is missing the `[Replaceable]` tag from the [specification](https://html.spec.whatwg.org/multipage/workers.html#dedicated-workers-and-the-dedicatedworkerglobalscope-interface):

```
 [Replaceable] readonly attribute DOMString name
```